openstack/compute/v2/extensions/simpletenantusage/results.go

```diff
@@ -0,0 +1,130 @@
+package simpletenantusage
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// TenantUsage is a set of usage information about a tenant over the sampling window
+type TenantUsage struct {
+	// ServerUsages is an array of ServerUsage maps
+	ServerUsages []ServerUsage `json:"server_usages"`
+
+	// Start is the beginning time to calculate usage statistics on compute and storage resources
+	Start string `json:"start"`
```

Comment:
This field should be parsed as `time.Time`. Based on the result I'm seeing, it'll need a custom unmarshal function. There are many of these throughout gophercloud to use as an example. It looks like the time format is a "MilliNoZ`, so look for an example that uses that format.